test/unit-tests/function/matrix/fft.test.js

```diff
@@ -0,0 +1,52 @@
+// test fft
+
+import approx from '../../../../tools/approx.js'
+import math from '../../../../src/defaultInstance.js'
+import { arraySize } from '../../../../src/utils/array.js'
+import { isMatrix } from '../../../../src/utils/is.js'
+const fft = math.fft
+
+function dft (arr) {
+  if (isMatrix(arr)) return math.matrix(dft(arr.valueOf()), arr.storage())
+  const size = arraySize(arr)
+  return math.matrixFromFunction(size, k => {
+    let acc = 0
+    math.matrix(arr).forEach((x, n) => {
+      acc = math.add(acc,
+        math.multiply(
+          math.exp(
+            math.multiply(
+              math.multiply(math.i, math.tau),
+              math.multiply(
+                -1,
+                math.dot(k, math.dotDivide(n, size))
+              )
+            )
+          ),
+          x
+        )
+      )
+    })
+    return acc
+  })
+}
+
+describe('inv', function () {
```

Comment:
typo: I guess `'inv'` should be `'fft'` here

Reply:
Done.